tests/system/test_produce_consume_commit_offset.py

```diff
@@ -37,8 +37,8 @@ def test_produce_many_consume_one(producer, consumer):
     with consumer:
         m = next(consumer)
         committed_offsets = consumer.commit(sync=True)
-        committed = list(filter(lambda tp: tp.partition == m.partition() and tp.topic == m.topic(), committed_offsets))[0]
-        assert committed.offset == m.offset() + 1
+        committed = list(filter(lambda tp: tp.partition == m.partition and tp.topic == m.topic, committed_offsets))[0]
+        assert committed.offset == m.offset + 1
```

Comment:
The `next()` function could be called, instead of `list()[0]`.

Like following:
`next(filter(lambda tp: tp.partition == m.partition and tp.topic == m.topic, committed_offsets))`

Reply:
Sure, you are right. There could be many improvements in many places, this one is not related to this particular PR :)

Reply:
Ok, sorry. Everything else seems good.

Reply:
You are more than welcome to get back with any feedback/suggestions! :)